flytekit/clients/raw.py

```diff
@@ -47,11 +47,11 @@ def _get_basic_flow_scopes() -> List[str]:
     deprecated_single_scope = _DEPRECATED_SCOPE.get()
     if deprecated_single_scope:
         return [deprecated_single_scope]
-    oauth_scopes = OAUTH_SCOPES.get()
-    if "openid" in oauth_scopes:
+    scopes = SCOPES.get()
```

Comment:
i think we may have to do something uglier like

```suggestion
    scopes = SCOPES.get() or OAUTH_SCOPES.get()
    if scopes and "openid" in scopes:
```